### DISCUSSION TITLE
Long running tasks in server/api

### DISCUSSION THREAD

**Discussion by Rigo-m** [nuxt/nuxt#16290] [Questions]
What if I need some server apis to be long running tasks (e.g.: workers unbound in cloudflare or background functions in netlify), how can I do it?

**Comment by wobsoriano** [comment]
You'll probably make a custom [nitro](https://nitro.unjs.io) preset for this

**Reply by Rigo-m** [reply]
Yes. I guess so. But it should be something handled at `routeRules` level, right? Because you dont want every route to be handled by a background function.
Paging @pi0 on this
